ror-tools/src/test/scala/tech/beshu/ror/tools/RorToolsAppSuite.scala

```diff
@@ -0,0 +1,231 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.tools
+
+import better.files.File as BetterFile
+import monix.execution.Scheduler
+import org.apache.commons.compress.archivers.tar.TarFile
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.matchers.must.Matchers.include
+import org.scalatest.matchers.should.Matchers.{equal, should, shouldNot}
+import org.scalatest.wordspec.AnyWordSpec
+import tech.beshu.ror.tools.RorToolsAppHandler.Result
+import tech.beshu.ror.tools.core.utils.InOut
+import tech.beshu.ror.tools.utils.{CapturingOutputAndMockingInput, DirectoryUtils, TestEsContainerManager}
+import tech.beshu.ror.utils.containers.*
+
+import java.io.{Console as _, *}
+import java.nio.file.{Files, Path}
+import scala.concurrent.duration.*
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+import scala.language.postfixOps
+
+class RorToolsAppSuite extends AnyWordSpec with BeforeAndAfterAll {
+
+  implicit val scheduler: Scheduler = Scheduler.computation(10)
+
+  private val tempDirectory = BetterFile.newTemporaryDirectory()
+  private val localPath = tempDirectory.path
+  private val esLocalPath = (tempDirectory / "es").path
+
+  // Before performing tests in this suite:
```

Comment:
if we add the `prepareElasticsearchAndRorBinaries` the comment should be defined close to it, not to the `beforeAll` method.